db/migrations/20230327090901_create_orders_table.sql

```diff
@@ -0,0 +1,22 @@
+-- migrate:up
+CREATE TABLE orders (
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  user_id INT NOT NULL,
```

Comment:
orders 테이블의 특성상 조회되는 경우를 생각해보면 bids 테이블에 buyer_id가 있긴 하지만 JOIN으로 인한 비용등을 고려한다면 orders 테이블 내에도 buyer_id, seller_id 둘 다 갖고 있는게 나을 것 같습니다. 둘 다 users table의 id를 foreign key로요.
```suggestion
  buyer_id INT NOT NULL,
  seller_id INT NOT NULL,
```